release-notes/release-6.19.md

```diff
@@ -23,14 +23,14 @@ Learn more in our [documentation](https://www.eclipse.org/che/docs/infra-support
 
 ---
 
-## Incorrect warning no longer emitted to the logs when workspace idling is disabled
+## Improvements in workspace idling diagnostics
 
-Eclipse Che improved on diagnostics of workspace activity and handling possible error conditions for
-workspace idling and expiration detection. Unfortunately, this caused an unnecessary warning to be
-emitted to the logs about missing workspace expiry time when workspace idling was disabled.
+Due to an improvement in Eclipse Che diagnostics of workspace activity, and error handling related to 
+and better handling of possible error conditions, we no longer log an unnecessary warning when
```

Comment:
`and error handling related to and better handling of possible error conditions` sounds weird

Reply:
Yes - this was a mistake.